src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -246,7 +246,7 @@ def init_sync_d_interface_tables(db_conn):
     # namespace to get a unique key. Assuming that ':' is not present in namespace
     # string or in sai id.
     # sai_id_key = namespace : sai_id
-    if_id_map = {get_sai_id_key(db_conn.namespace, sai_id): if_name for sai_id, if_name in if_id_map.items() if \
+    if_id_map = {get_sai_id_key(db_conn.namespace, sai_id): if_name.decode() for sai_id, if_name in if_id_map.items() if \
```

Comment:
>if_name.decode() [](start = 60, length = 16)

Many duplicated calls inside this function.

Reply:
Have updated the code to reduce the decode().
I kept the if_name_map to use bytes if_name, as the user functions use it as bytes  string.
But I had to add new dictionaries to fix this.